build/teamcity/cockroach/ci/tests/sqllogic_hi_vmodule_nightly_impl.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -xeuo pipefail
+
+bazel build //pkg/cmd/bazci --config=ci
+BAZEL_BIN=$(bazel info bazel-bin --config=ci)
+
+$BAZEL_BIN/pkg/cmd/bazci/bazci_/bazci --config=ci \
+    test //pkg/sql/logictest:logictest_test -- \
+    --test_arg=--vmodule=*=10 \
+    --test_arg=-show-sql \
+    --test_arg=-test.v \
+    --test_arg=-test.timeout=2h \
```

Comment:
Since you have `--test_timeout=7200` this shouldn't also be necessary.

Reply:
Done